src/types/GraphFilter.ts

```diff
@@ -6,6 +6,10 @@ export interface Duration {
   value: number;
 }
 
+export interface PollInterval {
```

Comment:
I think of `PollInterval` as being a `type` and not an interface since it doesn't have multiple properties inside of it.